src/common.ts

```diff
@@ -19,3 +19,4 @@ export const targetProps = [
   'owner',
   'branch',
 ];
+export const targetPropsWithId = [...targetProps, 'id'];
```

Comment:
`id` was dropped from targetProps to help match Gitlab projects but needed to send to import API when sanitizing